supporter-product-data/cloudformation/cfn.yaml

```
@@ -394,6 +394,26 @@ Resources:
       - SupporterProductDataQueue
       - ProcessSupporterRatePlanItemLambda
 
+  ExecutionFailureAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: CreateProdResources
+    Properties:
+      AlarmActions:
+        - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:reader-revenue-dev
+      AlarmName: !Sub Supporter Product Data step function Failure in ${Stage}
+      AlarmDescription: !Sub The supporter-product-data-${Stage} step function has failed. Check https://eu-west-1.console.aws.amazon.com/states/home?region=eu-west-1#/statemachines/view/arn:aws:states:eu-west-1:865473395570:stateMachine:supporter-product-data-PROD?statusFilter=FAILED for details.
```

There's hardcoded stage in the AWS ARN in the URL:
```suggestion
      AlarmDescription: !Sub The supporter-product-data-${Stage} step function has failed. Check https://eu-west-1.console.aws.amazon.com/states/home?region=eu-west-1#/statemachines/view/arn:aws:states:eu-west-1:865473395570:stateMachine:supporter-product-data-${STAGE}?statusFilter=FAILED for details.
```

---

Yeah this alarm only exists in PROD but this is still better 👍